common/gossmods_listpeerchannels.c

```diff
@@ -109,6 +114,8 @@ gossmods_from_listpeerchannels_(const tal_t *ctx,
 				JSON_SCAN(json_to_int, &scidd.dir),
 				JSON_SCAN(json_to_msat, &spendable),
 				JSON_SCAN(json_to_msat, &receivable),
+				JSON_SCAN(json_to_msat, &max_total_in_htlc),
+				JSON_SCAN(json_to_msat, &max_total_out_htlc),
```

Comment:
how often does this get updated?

Reply:
The `max_htlc_value_in_flight` pair is fixed for the entire lifetime of the channel.
The `gossmod_from_listpeerchannels` is called to parse the response from `listpeerchannels`.
In `askrene` or `renepay` this is expected to happen once per rpc call.